.travis.yml

```diff
@@ -1,22 +1,10 @@
 language: elixir
 sudo: false
 elixir:
-  - 1.4
```

Comment:
What happens if people are using the now-unsupported versions of Elixir? Do they get an error when running `mix deps.get`? Or does the code just break?

Reply:
If my understanding of hex dependency resolution is correct, because the package declares the minimum version they'll be stuck at a `0.10.X` version and unable to move to `>= 0.11`.